src/scripts/main.js

```diff
@@ -23,6 +23,26 @@ const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
   // WRITE YOUR CODE HERE
+
+  for (const key in data) {
+    const ul = document.createElement('ul');
+    const li = document.createElement('li');
+
+    element.append(ul);
+    ul.append(li);
+
+    li.textContent = key;
+
+    for (const item in createTree(li, data[key])) {
+      const ul2 = document.createElement('ul');
```

Comment:
you're doing the same as above (DRY)
overwrite the function to avoid duplicating a code

just do check: if there are keys in your element, call `createTree` for this element again